### ISSUE TITLE
Global tags dupplication

### ISSUE THREAD

**Issue by evilz** [FastEndpoints/FastEndpoints#654]
Adding global tag make it repeat many time. One for each endpoint.

**Code**

```csharp
services.SwaggerDocument(o =>
        {
 o.TagDescriptions = t => t["Partners"] = "Manage partners";
}
```

Result in Swagger.json
```json
{
 "tags": [
    {
      "name": "Partners",
      "description": "Manage partners"
    },
    {
      "name": "Partners",
      "description": "Manage partners"
    },
    {
      "name": "Partners",
      "description": "Manage partners"
    },
    {
      "name": "Partners",
      "description": "Manage partners"
    }
  ]
}
```

**Comment by evilz** [comment]
I guess the issue is here 
https://github.com/FastEndpoints/FastEndpoints/blob/103e4c4a41a2564f30b7295c2054fc169b92a1bb/Src/Swagger/Extensions.cs#L71-L77

Maybe just check if tag key is already in collection.

Can I do a PR ?

**Comment by KateyBee** [comment]
Thanks for reporting the issue. It has been solved in `v5.23.0.14-beta`.
Went in a different route than checking for duplicates:

https://github.com/FastEndpoints/FastEndpoints/blob/6aec23783b1cf3a13dd4fb4e3c734e48ddf72dd8/Src/Swagger/Extensions.cs#L66-L78
